uiplib/gui/minimizer.py

```diff
@@ -0,0 +1,71 @@
+from gi.repository import Gtk as gtk , AppIndicator3 as appindicator , Notify as notify
+import os
+from uiplib.gui.mainGui import MainWindow
+
+APPINDICATOR_ID = 'myappindicator'
+
+#Notifications
+
+notify_prev= notify.Notification.new("<b>UIP</b>",'Applying Previous Wallpaper', os.path.abspath('uip_icon.svg'))
+notify_next= notify.Notification.new("<b>UIP</b>",'Applying Next Wallpaper', os.path.abspath('uip_icon.svg'))
+
+
+class minimizer:
+	
+	def __init__(self, settings, wallpaper):
+		self.wallpaper = wallpaper
+		self.settings = settings
+	#Menu items
+
+	def quit(self, source):
+		notify.uninit()
+		gtk.main_quit()
+
+	def prev_(self, source):
+		notify_prev.show()
+		app = MainWindow(self.settings, self.wallpaper)
+		app.prev_wallpaper()
+		app.set_wallpaper()
+		print('Prev image')
+
+	def next_(self, source):
+		notify_next.show()
+		app = MainWindow(self.settings, self.wallpaper)
+		app.next_wallpaper()
+		app.set_wallpaper()
+		print('Next image')
+
+	def maximize(self, source):
+		print('Maximizing UIP')
+		app = MainWindow(self.settings, self.wallpaper)
+		app.run()
+
+	#Building Menu
+
+	def build_menu(self):
+		self.menu = gtk.Menu()
+		self.item_next = gtk.MenuItem('Next')
+		self.item_next.connect('activate', self.next_ )
+		self.menu.append(self.item_next)
+
+		self.item_prev = gtk.MenuItem('Previous')
+		self.item_prev.connect('activate', self.prev_ )
+		self.menu.append(self.item_prev)
+
+		self.item_max = gtk.MenuItem('Maximize')
+		self.item_max.connect('activate', self.maximize )
+		self.menu.append(self.item_max)
+
+		self.item_quit = gtk.MenuItem('Quit')
+		self.item_quit.connect('activate', quit)
+		self.menu.append(self.item_quit)
+
+		self.menu.show_all()
+		return self.menu
+
+	def run(self):
+		self.indicator = appindicator.Indicator.new(APPINDICATOR_ID, os.path.abspath('uip_icon.svg'), appindicator.IndicatorCategory.SYSTEM_SERVICES)
+		self.indicator.set_status(appindicator.IndicatorStatus.ACTIVE)
+		self.indicator.set_menu(self.build_menu())
+		notify.init(APPINDICATOR_ID)
+		gtk.main()
```

Comment:
okay. Will change these, once I get the prev and next function working, i.e. before updating this PR